frontend/app/routes/$lang+/_protected+/letters+/index.tsx

```diff
@@ -105,6 +106,7 @@ export default function LettersIndex() {
             <li key={letter.id} className="py-4 sm:py-6">
               <InlineLink reloadDocument to={`/letters/${letter.id}/download`} className="external-link font-lato font-semibold" target="_blank">
                 {getNameByLanguage(i18n.language, letterType)}
+                <NewTabIndicator>{t('gcweb:screen-reader.newtab')}</NewTabIndicator>
```

Comment:
Move the text translation inside the `NewTabIndicator` component, otherwise that component is basically useless.